php_memcached.c

```diff
@@ -4049,12 +4045,12 @@ ZEND_BEGIN_ARG_INFO(arginfo_setOption, 0)
 ZEND_END_ARG_INFO()
 
 ZEND_BEGIN_ARG_INFO(arginfo_setOptions, 0)
-	ZEND_ARG_INFO(0, options)
+	ZEND_ARG_ARRAY_INFO(0, options, 0)
 ZEND_END_ARG_INFO()
 
 ZEND_BEGIN_ARG_INFO(arginfo_setBucket, 3)
-	ZEND_ARG_INFO(0, host_map)
-	ZEND_ARG_INFO(0, forward_map)
+	ZEND_ARG_ARRAY_INFO(0, host_map, 0)
+	ZEND_ARG_ARRAY_INFO(0, forward_map, 0)
```

Comment:
Sorry for not getting back sooner. I've tried to build the lib. on a PHP7 box but I didn't get it to work so I haven't debug it yet.
However looking at `Memcached::setBucket(array host_map, array forward_map, integer replicas)` it makes sense that both the `host_map` and `forward_map` must be an array because;
- `host_map` must be an array
- `forward_map` must be of the same size as `host_map`
- i.e. `forward_map` must be an array as well